### PR TITLE
fix: prefer tfsec-bin AUR package

### PR DESCRIPTION
**Pull request by greut**
Building from the sources is not the preferred way and tfsec-bin uses the binary provided by this project (and checks the signature)

- https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tfsec
- https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tfsec-bin

### CODE REVIEW

**Review by liamg — APPROVED** [review]

**Comment by liamg** [comment]
Thanks @greut - is it possible for you to add [me](https://aur.archlinux.org/account/liamg) as a maintainer?

**Comment by greut** [comment]
Of course.  On Thu, Jun 16, 2022 at 18:17, Liam Galvin ***@***.***> wrote:  
Thanks @greut - is it possible for you to add me as a maintainer?

**Comment by greut** [comment]
@liamg I gave a look at https://goreleaser.com/customization/aur/ but it doesn't seem to be applicable straight away.

https://aur.archlinux.org/packages/tfsec-bin is done, https://aur.archlinux.org/packages/tfsec will have to wait for the actual maintainer to answer my request :)

Happy packaging!

**Comment by greut** [comment]
@liamg fyi https://github.com/arch4edu/aur-auto-update/pull/16

**Pull request merged**
